src/translations/en.json

```diff
@@ -257,8 +257,17 @@
   "country": "Country",
   "voltageRegulatorOn": "Voltage regulation",
   "PlannedActivePowerSetPoint": "Planning active power set point",
-  "downloadCases": "Download Case(s)",
-
+  "download.button": "Download",
```

Comment:
Yes, there should be work done with the application's translation keys, they are over the place. Because I work on another ticket modifying them, I'd like to structure them better, and we have to start somewhere (so, let's start here).